.azure-pipelines.yml

```diff
@@ -166,6 +192,22 @@ jobs:
   - task: DownloadPipelineArtifact@1
     inputs:
       targetPath: $(Build.ArtifactStagingDirectory)
+  - script: |
+      set -e
+      mv $(Build.ArtifactStagingDirectory)/doc_book gh-pages
+      mv $(Build.ArtifactStagingDirectory)/doc_api gh-pages/api
+    displayName: Create gh-pages directory
+  - script: |
+      set -e
+      cat .git/config
```

Comment:
This seems like it might be a debugging leftover? If it's actually needed for some weird reason, please add a comment!

Reply:
Oops definitely not needed!